runner/ansible/vars/azure_env.yml

```diff
@@ -10,4 +10,11 @@ expected:
   "1.1.7": "2"
   "1.1.8": "1"
   "1.2.1": "true"
+  "1.2.2.sbd": "144"
+  "1.2.2.fence_azure_arm": "900"
+  "1.3.1": "yes"
+  "1.3.2": "always"
+  "1.3.3": "enabled"
```

Comment:
We don't really need this `1.3.3` expected value, right?

Reply:
No we do not.